cypress/integration/Dictionary/createDictionary.feature

```diff
@@ -0,0 +1,110 @@
+Feature: Dictionary Management
+
+  Scenario: A user to create a new dictionary
+    Given the user is logged in
+    And is on the user collections page
+    Then the user clicks on Create Dictionary Icon
+     Then the user should be able to create new dictionary
+
+  Scenario: A user should be able to create new dictionary by pressing on Create new dictionary icon
+    Given the user is on the Dictionaries page
+     When the user clicks on Create new dictionary icon
+     Then the create Dictionary form is filled
+     Then the user should be able to create new dictionary
+  
+  Scenario: A user should be able to create a new dictionary by filling the required information
+    Given the user is on the Create Dictionary page
+     When the user enters the required information in the form
+      And the user clicks the submit button
+     Then the user should be able to create a new dictionary
+  
+  Scenario: The Create Dictionary form should require a Dictionary Name to Create new dictionary
+    Given the user is on the Create Dictionary page
+     When the user enters their Dictionary Name
+      And the user submits the form
+     Then the user should still be on the Create Dictionary page
+      And the Short code, Preferred Source, Owner, Visibility and Preferred Language fields should be marked as having an error
+      And the error message "Short code is required" should be visible
+      And the error message "Select a preferred source" should be visible
+      And the error message "Select this dictionary's owner" should be visible
+      And the error message "Select who will have access to this dictionary" should be visible
+      And the error message "Select a preferred language" should be visible
+  
+  Scenario: The Create Dictionary form should require a Short Code to Create new dictionary
```

Comment:
So a word on capitalization: I think it's fine to describe pages and forms with capitals because when you go the the "create dictionary" page, it says "Create Dictionary" on the top. Otherwise, capitalisation should generally be avoided except where, e.g., at the start of a scenario (which is really a sentence).

```suggestion
  Scenario: The Create Dictionary form should require a short sode to create new dictionary
```

Reply:
@ibacher thanks for the review! Let me fix the proposed changes.